openquake/hazardlib/gsim/yu_2013.py

```diff
@@ -0,0 +1,242 @@
+    DEFINED_FOR_INTENSITY_MEASURE_COMPONENT = const.IMC.AVERAGE_HORIZONTAL
+
+    #: Supported standard deviation types is total
+    DEFINED_FOR_STANDARD_DEVIATION_TYPES = set([
+        const.StdDev.TOTAL
+    ])
+
+    #: No site parameters required
+    REQUIRES_SITES_PARAMETERS = set(())
+
+    #: Required rupture parameter is magnitude
+    REQUIRES_RUPTURE_PARAMETERS = set(('mag',))
+
+    #: Required distance measures are epicentral distance and azimuth
+    REQUIRES_DISTANCES = set(('repi', 'azimuth'))
+
+    def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
+            """
+            See :meth:`superclass method
+            <.base.GroundShakingIntensityModel.get_mean_and_stddevs>`
+            for spec of input and result values.
+            """
+            # Check that the requested standard deviation type is available
+            assert all(stddev_type in self.DEFINED_FOR_STANDARD_DEVIATION_TYPES
+                       for stddev_type in stddev_types)
+            #
+            # Set coefficients
+            C = self.COEFFS[imt]
+            if rup.mag > 6.5:
+                a1ca = C['ua']
+                a1cb = C['ub']
+                a1cc = C['uc']
+                a1cd = C['ud']
+                a1ce = C['ue']
+                a2ca = C['ia']
+                a2cb = C['ib']
+                a2cc = C['ic']
+                a2cd = C['id']
+                a2ce = C['ie']
+            else:
+                a1ca = C['a']
+                a1cb = C['b']
+                a1cc = C['c']
+                a1cd = C['d']
+                a1ce = C['e']
+                a2ca = C['ma']
+                a2cb = C['mb']
+                a2cc = C['mc']
+                a2cd = C['md']
+                a2ce = C['me']
+            #
+            # Compute the mean value (i.e. natural logarithm of ground motion)
+            mag = rup.mag
+            epi = dists.repi
+            theta = dists.azimuth
+            #
+            # Get correction coefficients
+            ras = []
+            for epi, theta in zip(dists.repi, dists.azimuth):
+                res = minimize(fnc, epi, args=(epi, theta, mag, C),
+                               method='Nelder-Mead', tol=0.1)
+                ras.append(res.x[0])
+            ras = np.array(ras)
+            rbs = rbf(ras, C, mag)
+            #
+            # Compute values of ground motion for the two cases
+            mean1 = (a1ca + a1cb * mag + a1cc *
+                     np.log((ras**2+225)**0.5 + a1cd * np.exp(a1ce * mag)))
+            mean2 = (a2ca + a2cb * mag + a2cc *
+                     np.log((rbs**2+225)**0.5 + a2cd * np.exp(a2ce * mag)))
+            #
+            # Get distances
+            x = (mean1 * np.sin(np.radians(dists.azimuth)))**2
+            y = (mean2 * np.cos(np.radians(dists.azimuth)))**2
+            mean = mean1 * mean2 / np.sqrt(x+y)
+            if isinstance(imt, (PGA)):
+                mean = np.exp(mean)/g/100
```

Comment:
Same PEP 8 violations here

Reply:
After the fix above I cannot find other PEP8 violations (also using this http://pep8online.com/)